wpiunits/src/main/java/edu/wpi/first/units/Measure.java

```diff
@@ -105,6 +108,10 @@ default <U2 extends Unit<U2>> Measure<?> times(Measure<U2> other) {
         && unit().getBaseUnit().equals(((Per<?, ?>) other.unit()).denominator().getBaseUnit())) {
       Unit<?> numerator = ((Per<?, ?>) other.unit()).numerator();
       return numerator.ofBaseUnits(baseUnitMagnitude() * other.baseUnitMagnitude());
+    } else if (other.unit() instanceof Velocity && unit().getBaseUnit().equals(Seconds)) {
```

Comment:
use instanceof pattern matching to avoid the cast on line 113

Reply:
This was copy-pasted from lines 103-106, just flipping `this` and `other`. I can fix all of them to use `instanceof` if that's what we want, though.

Reply:
For now, just use instanceof pattern matching on the lines you wrote. I'm doing the rest over at #6691, so let's try to reduce merge conflicts later.